distribution/ddf-common/src/main/resources/bin/get_property

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+PROPERTY_NAME=$1
+
+# Useful directories
+SCRIPTDIR=$(dirname $0)
+HOME_DIR=$(cd "${SCRIPTDIR}/.."; pwd -P)
+
+# Useful files
+PROPERTIES_FILE=$HOME_DIR/etc/system.properties
+
+# Check file exists where it is supposed to
+if [ ! -f $PROPERTIES_FILE ]; then
+    echo Cannot find file $PROPERTIES_FILE >&2
+    exit 1
+fi
+
+# Extract the value of a property from a Java properties file
+# This function does not handle multi-line properties
+# If the property does not exist, print  error message to stderr
+value=""
```

Comment:
won't need this if you using the `-z` operator in the `if` bracket

Reply:
>Just out of curiosity why are we not using the value of the solr.http.url to determine whether or not we need to start solr?

I don't have an answer. I'm hesitant to do that because every time I think I can predict when to start/stop Solr automatically, reality proves me wrong. Let me tag @mcalcote and @rzwiefel on it so that can weigh in.